hail/src/main/scala/is/hail/expr/types/physical/PContainer.scala

```diff
@@ -127,6 +136,17 @@ abstract class PContainer extends PType {
     region.allocate(contentsAlignment, contentsByteSize(length))
   }
 
+  def setAllMissingBits(region: Region, aoff: Long, length: Int) {
+    if (elementType.required)
+      return
+    val nMissingBytes = (length + 7) / 8
+    var i = 0
```

Comment:
Same comment about utilizing common code. Perhaps a helper function that takes the byte to store?

Reply:
done